modes/micromobility.md

```diff
@@ -37,9 +45,24 @@ The `trip_attributes` array is not used with the micromobility mode.
 The `vehicle_attributes` array may have the following key value pairs:
 
 - `year` (integer)
-- `manufacturer` (string)
+- `make` (string)
```

Comment:
Are you changing this just because this is standard terminology for cars? I know 2.0 is breaking but should it stay `manufacturer` for consistency? Just checking, but I do like using `make` better I believe.

Reply:
Per @marie-x, "I wanted to do make and model because that seems more standard"